site/_docs/reference.md

```diff
@@ -1447,10 +1448,10 @@ period:
 | {fn SUBSTRING(string, offset, length)} | Returns a character string that consists of *length* characters from *string* starting at the *offset* position
 | {fn UCASE(string)} | Returns a string in which all alphabetic characters in *string* have been converted to upper case
 | {fn REPLACE(string, search, replacement)} | Returns a string in which all the occurrences of *search* in *string* are replaced with *replacement*; if *replacement* is the empty string, the occurrences of *search* are removed
+| {fn ASCII(string)} | Returns the corresponding ASCII code of the first character of *string*; Returns 0 if *string* is empty; Returns NULL if *string* is NULL
```

Comment:
Let's agree on behaviour of `ASCII` function for non-basic ASCII characters (> `0x7E`).  This includes extended ASCII (eg. LATIN) and UNICODE.

Should we also support more generic [UNICODE](https://docs.microsoft.com/en-us/sql/t-sql/functions/unicode-transact-sql?view=sql-server-2017) function ?

Reply:
In my opinion,  explicitly saying that behavior is undefined for non-basic ascii looks like fine.